config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Refinery::Core::Engine.routes.draw do
   namespace :blog, :path => Refinery::Blog.page_url do
     root :to => "posts#index"
-    resources :posts, :only => [:show]
+    resources :posts, path: '', :only => [:show] do
```

Comment:
> Was something like:
http://localhost:3000/blog/posts/blog-post-name
Now it is:
http://localhost:3000/blog/blog-post-name

If I've an existing blog indexed on google, all indexed posts will now point to a 404.

I suggest to add a permanent redirect from `blog/posts/*` to blog/*` in order to keep SEO index fine.